Tools/peg_generator/pegen/grammar_parser.py

```diff
@@ -2,7 +2,7 @@
 # @generated by pegen from ./Tools/peg_generator/pegen/metagrammar.gram
 
 import ast
-from typing import Optional
+from typing import Optional, Any
```

Comment:
Doesn't look like this is needed

Reply:
It is needed, otherwise the generated parsers will fail to get the references to `Any`. Try to remove it and you will see the error I am referring to.